requests/api.py

```diff
@@ -33,7 +33,7 @@ def request(method, url, **kwargs):
         before giving up, as a float, or a :ref:`(connect timeout, read
         timeout) <timeouts>` tuple.
     :type timeout: float or tuple
-    :param allow_redirects: (optional) Boolean. Enable/disable GET/OPTIONS/POST/PUT/PATCH/DELETE/HEAD redirection. Defaults to ``True``.
+    :param allow_redirects: (optional) Boolean. Enable/disable GET/OPTIONS/POST/PUT/PATCH/DELETE/HEAD redirection. Defaults to ``False`` for HEAD, ``True`` for all others.
     :type allow_redirects: bool
```

Comment:
This is not correct. `requests.head` defaults this to `False`, but all other uses (including `requests.request('HEAD', ...)` default it to `True`.

Reply:
OK, understood- and sorry for the delay, busy week. I'll look to get this updated before the end of the weekend.

Reply:
No worries: we're all busy people!